examples/jsm/math/MeshSurfaceSampler.js

```diff
@@ -23,12 +23,6 @@ class MeshSurfaceSampler {
 
 		let geometry = mesh.geometry;
 
-		if ( ! geometry.isBufferGeometry || geometry.attributes.position.itemSize !== 3 ) {
```

Comment:
Why have you removed the check fir `geometry.attributes.position.itemSize !== 3`?

Reply:
I think the item size of the position attribute is always 3 (otherwise it should be invalid)? But I can return this check.

Reply:
No, it's fine. We do indeed assume at other places that the item size is always three, so this check was overcautious.